rimage/file_simple.c

```diff
@@ -568,3 +568,26 @@ const struct adsp machine_imx8 = {
 	.machine_id = MACHINE_IMX8,
 	.write_firmware = simple_write_firmware,
 };
+
+const struct adsp machine_imx8x = {
```

Comment:
Do you mind also doing this patch for the rimage repo too. We need to keep aligned before we switch over.

Reply:
@dbaluta Do we nee to update CI too for the new platforms ?

Reply:
@lgirdwood all right will put this patch to rimage repo too. Yes, will need to update CI too. I will contact @xiulipan for this.

Reply:
@lgirdwood See https://github.com/thesofproject/rimage/pull/2 for rimage PR.

Reply:
@dbaluta Do we have the scripts for the simple build?
If not I will have a full PR set to add both thing after my local test.

Reply:
@xiulipan See PR https://github.com/thesofproject/sof/pull/2288/commits/f1578035d3151cdfc470d6c05d15bc51a1119e6d